mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -620,6 +627,7 @@ CLASS net/minecraft/unmapped/C_triydqro net/minecraft/block/AbstractBlock
 			COMMENT <p>This also marks a block as non-opaque.
 		METHOD m_dsszzhch pistonBehavior (Lnet/minecraft/unmapped/C_mojvzhua;)Lnet/minecraft/unmapped/C_triydqro$C_xnkxsdfy;
 			ARG 1 pistonBehavior
+		METHOD m_dyoolquj noBlockDustParticles ()Lnet/minecraft/unmapped/C_triydqro$C_xnkxsdfy;
```

Comment:
```suggestion
		METHOD m_dyoolquj withoutBlockDustParticles ()Lnet/minecraft/unmapped/C_triydqro$C_xnkxsdfy;
```
[CONVENSION.md "withers"](https://github.com/QuiltMC/quilt-mappings/blob/1.20-pre6/CONVENTIONS.md#getters-setters-withers-and-creators)
> Use "with" for methods that return a copy of an object with a different value for some property. Name the parameter the same as the property.

Reply:
oh btw this name that I've filled followed a convention that was already in place on the Settings class; however, I like the idea of `without`! We should investigate further